schema/table.go

```diff
@@ -198,6 +197,7 @@ func (t Table) Resolve(ctx context.Context, meta ClientMeta, syncTime time.Time,
 				}
 			}
 
+			totalResources++
```

Comment:
It seems more clear to me to count each resource at a time when we send it to the channel instead of the current approach. Happy to revert as this is a personal opinion